frame/bridge/relayer-game/src/lib.rs

```diff
@@ -575,9 +576,8 @@ impl<T: Config<I>, I: Instance> Module<T, I> {
 
 							if previous_maybe_extended_relay_affirmation_id.is_none() {
 								let relay_header_parcels = &round_relay_affirmations
-									.into_iter()
-									.nth(index as usize)
-									.unwrap()
+									.get(index as usize)
+									.unwrap() // TODO: Remove this unwrap
```

Comment:
@AurevoirXavier Do we need to deal with this?

Reply:
relayer game will be removed while eth2.0 released.